integration/specs/CarouselCard/carouselCard-3.spec.js

```diff
@@ -0,0 +1,28 @@
+const PageCarouselCard = require('../../../src/components/CarouselCard/pageObject');
+const { ARROW_RIGHT_KEY } = require('../../constants');
+
+const CAROUSEL = '#carousel-3';
+
+const addNewCard = () => $('#button-icon_add-new-card').click();
+
+describe('CarouselCard with CarouselImagen changed dynamically', () => {
+    it('should select the new option with keyboard after it is added dynamically', () => {
+        browser.url('/#!/CarouselCard/3');
+        const component = $(CAROUSEL);
+        component.waitForExist();
+        var carousel = new PageCarouselCard(CAROUSEL);
```

Comment:
Let's use const instead, we never use var, when the variable is not reassigned use const otherwise use let.